src/StarkEx.Crypto.SDK/Encoding/AssetEncoder.cs

```diff
@@ -202,6 +218,53 @@ private static string GetMintableErc20AssetId(string mintingBlob, string address
             .And(MintableAndBitMask)
             .Or(MintableOrBitMask);
 
+        keccackHashAsBigInteger = keccackHashAsBigInteger
+            .ClearBit(240)
+            .ClearBit(241)
+            .ClearBit(242)
+            .ClearBit(243)
+            .ClearBit(244)
+            .ClearBit(245)
+            .ClearBit(246)
+            .ClearBit(247)
+            .SetBit(248)
+            .SetBit(249)
+            .SetBit(250);
```

Comment:
I'm unsure if these new encodings should always be applied or should only be applied when mint amounts are bigger than one. Will ask StarkEx team about this

Reply:
It's not clear on the docs, thanks for asking them!